### ISSUE TITLE
9/5 Authorship Scorecard

### ISSUE THREAD

**Issue by saveth** [lzim/teampsd#425]
**BEFORE FILLING OUT THIS TEMPLATE, PLEASE ASSIGN THIS ISSUE TO THE BUG KANBAN UNDER PROJECTS (YES, EVEN IF IT IS A FEATURE. ALL ISSUES START IN THE BUG KANBAN FOR TRIAGE). --------------------->**

**Describe the opportunity**
Create a ShinyApp that enables the team to score and rank team members for various papers.

**FEATURE**

**7. How would you like this feature to work?**
ShinyApp should be able to allow users to allocate score for various components of a project, contribution to writing the paper, etc. and generate a final score and authorship rank based on input scoring.

**8. How urgent is this request? (Urgent/Not Urgent)**
not urgent

**9. Why is this feature important to you?**
Allows the team to see how authorship scoring/ranking in a transparent manner.

**10. Is your feature request related to a problem? Please describe.**

**11. Is there a deadline or other scheduled constraint driving your request?"**

**12. Additional context**

**STOP HERE. INFORMATION BELOW IS TO BE FILLED OUT BY WORKGROUPS AS PART OF TRIAGE AND WORK BREAKDOWN PROCESS**

**Estimated person hours to complete:**

**Key people (use @ assignment for people whose input will be necessary):**
@saveth @lzim 

**Key dependencies:**
Are there other tasks that need to be completed first?

**Constraints:**
Is there a resource constraint that may affect the development of the feature?

**Gantt chart needed? Attach link and/or screenshot.**

**Comment by lzim** [comment]
Hi @saveth 

I was attempting to test today, and we ran into an issue: https://github.com/lzim/teampsd/issues/459
I wasn't sure whether to just discuss it here, or to create a new issue.

Thanks!
@staceypark @jessfroe

**Comment by saveth** [comment]
Pushed another version of the Authorship Scorecard to ShinyApps. Not experiencing any disconnection.

**Comment by lzim** [comment]
@saveth 

The Authorship page here: https://savet.shinyapps.io/Authorship/
Need the following edits:

- Remove: "2. Select Overall Eligible Weight Relative to Responsible."
- Edit to be 2. "Enter Weights for Each Authorship Eligible Contribution"
- Keep the following 6 categories only for Authorship Eligible Contributions:
1. Research Design/Procedures/Grant
2. Qualitative Data Analysis & Conceptualization
3. Quantitative Data Analysis & Conceptualization
4. Iterative SD Model Development/Validation
5. Facilitation theory of change and Facilitation resources
6. Writing Responsibilities

- We need a note that says: "Writing responsibilities should be set to 20% unless otherwise directed by the P.I. (Lindsey)."

**Comment by lzim** [comment]
@saveth @staceypark @jessfroe 

I will reorganize the contributions on OSF to reflect the 6 major author eligible contributions.
- I will let you know how the weights will sum to 100 within the 6 categories.
- We will review at a future Tuesday Manuscript meeting.

Note: There may be some specific manuscripts where we will focus on some of the other categories, so we may need to keep them as optional selections in the app.
 
> The Authorship page here: https://savet.shinyapps.io/Authorship/
> Need the following edits:
> 
> * Remove: "2. Select Overall Eligible Weight Relative to Responsible."
> * Edit to be 2. "Enter Weights for Each Authorship Eligible Contribution"
> * Keep the following 6 categories only for Authorship Eligible Contributions:
> 
> 1. Research Design/Procedures/Grant
> 2. Qualitative Data Analysis & Conceptualization
> 3. Quantitative Data Analysis & Conceptualization
> 4. Iterative SD Model Development/Validation
> 5. Facilitation theory of change and Facilitation resources
> 6. Writing Responsibilities
> 
> * We need a note that says: "Writing responsibilities should be set to 20% unless otherwise directed by the P.I. (Lindsey)."

**Comment by saveth** [comment]
@lzim Okay, I'll look for the updates once you've finished editing the categories on OSF/

**Comment by staceypark** [comment]
@lzim @saveth @jessfroe should we review this again at the 9/3 manuscript meeting along with the mentee & gmb papers?

**Comment by lzim** [comment]
> @lzim @saveth @jessfroe should we review this again at the 9/3 manuscript meeting along with the mentee & gmb papers?

No. Priority is the proportion of CBOC and Medical Center analyses for site selection and the two manuscript we need out the door!

**Comment by staceypark** [comment]
@lzim please provide feedback on this before the 9/5 quant meeting for review during meeting

**Comment by lzim** [comment]
FYI HQ:

From yesterday’s Intersession Prep:

- [ ] Need to put the training schedule prominently in the **team_tracker**

- [ ] Need to link issue #303 in the top of each _MTL Facilitate_ Training

- [ ] Re-name the 3 weekly Facilitate Sessions (M/W/F) according to what will be covered that week (s07: _Modeling to Learn_ base case)

- [ ] Cancel next week’s meetings, BUT...

- [ ] 1. First, back-up the video session from yesterday in 000 and link directly to it in the cancellation message

- [ ] 2. Second, put the updated video links in the **depend_products** column. 

- [ ] 2. Third, be sure to say in the email: “THIS IS A ONE-TIME CANCELLATION due to many team members being out for travel this week. 

Thanks! 🤓

**Comment by saveth** [comment]
@staceypark requested that the Responsible section in the ShinyApp be updated to reflect the columns in the manuscript trackers (https://github.com/lzim/teampsd/projects/9) and subsection listed under the cards.

FYI: @saveth @ritahitching @lzim

**Comment by saveth** [comment]
@staceypark The responsible section of the Authorship app (https://savet.shinyapps.io/Authorship/) has been updated to reflect the sections in the manuscript tracker.

FYI: @saveth @ritahitching @lzim

**Comment by lzim** [comment]
Great, thanks @saveth !!!

@staceypark Would you or @ritahitching take a look at this and see if it's ready for my review?

Then, we need to have Yogita help us get this on the Team PSD Shiny Server, so that it doesn't create too much demand on Savet's personal account.

**Comment by ritahitching** [comment]
Issue not ready to be closed - @lzim please update any remaining issues with the aim to close the issue.

**Comment by saveth** [comment]
Per meeting on 12/12 (https://meet.lucidmeetings.com/meeting/235910), the following updates are needed:

- Rename the first tab to say Name

- Rename the third tab to say Contribution

- Rename the fourth tab to say Order

- Rename the fifth tab to say Download and add instructions to "Bring this to the next manuscript and check this off as completed in the mtl.how/manuscripts tracker."

- Number the tabs (1. Name, 2. Manuscript, 3. Contribution, 4. Order, 5. Download)

- Add instructions on the first/home page with instructions that outline it's a 5 step process that must be done in order and add instructions of what needs to be completed in each step.

FYI: @saveth @staceypark @ritahitching @lzim

**Comment by saveth** [comment]
@staceypark I've made the changes we've discussed a couple weeks ago regarding the authorship app. Please copy the ui.R file onto your computer and push the changes to the teampsd shiny account.

FYI: @saveth @lzim

**Comment by staceypark** [comment]
@staceypark give @lzim link to TeamPSD authorship app to make the shortlink

**Comment by staceypark** [comment]
@lzim can we close this?
